console/console-init/ui/src/Tests/LinkList.test.tsx

```diff
@@ -6,7 +6,10 @@
 import React from "react";
 import { render } from "@testing-library/react";
 import { MemoryRouter } from "react-router";
-import { ILink, LinkList } from "modules/connection-detail/LinkList";
+import {
+  ILink,
+  ConnectionLinksList
+} from "modules/connection-detail/components/ConnectionLinksList/ConnectionLinksList";
```

Comment:
as we are following standard, component folder must have index.ts file and export in components index.ts file and don't have complete import path.
you can have either component folder level or component only not till component file. It will in future if your file move and folder name change , it will not cause the issue where you are importing that component because you component exposing at level of components folder.

import {
  ILink,
  ConnectionLinksList
} from "modules/connection-detail/components";